src/analysis/convex/cone/proper.lean

```diff
@@ -15,10 +15,11 @@ linear programs, the results from this file can be used to prove duality theorem
 
 ## TODO
 
-In the next few PRs (already sorry-free), we will add the definition and prove several properties
-of proper cones and finally prove the cone version of Farkas' lemma (2.3.4 in the reference).
+In the next few PRs (already sorry-free), we will prove the cone version of Farkas' lemma (2.3.4 in
+the reference).
```

Comment:
```suggestion
Prove the cone version of Farkas' lemma (2.3.4 in the reference).
```
"next few PRs" is helpful in a PR description, but not in the module comment. Imagine if mathlib is frozen and this comment is here forever!